frontend/src/styles/GlobalStyles.ts

```diff
@@ -37,9 +37,10 @@ export const GlobalStyles = createGlobalStyle`
     box-sizing: border-box;
   }
 
-html {
+  html:has(.burger-menu) {
```

Comment:
Isn't common make a individual configuration(burger-menu component) in a global configuration file.

Reply:
I put this for disable vertical scroll in burger menu

Reply:
i could try to implement this in another place